kube-flagd-proxy/main.go

```diff
@@ -1,17 +1,32 @@
 package main
 
 import (
-	"log"
+	"bytes"
 
-	"github.com/open-feature/flagd/core/pkg/logger"
-	"go.uber.org/zap/zapcore"
+	"github.com/dimiro1/banner"
+	"github.com/mattn/go-colorable"
+	"github.com/open-feature/flagd/kube-flagd-proxy/cmd"
 )
 
-func main() {
-	l, err := logger.NewZapLogger(zapcore.DebugLevel, "console")
-	if err != nil {
-		log.Fatalf("initialize zap logger: %v", err)
-	}
+var (
+	version    = "dev"
+	commit     = "HEAD"
+	date       = "unknown"
```

Comment:
Should these values be hardcoded?

Reply:
yes, they are overwritten on build there are the default values - mirrors the build command used in flagd

https://github.com/open-feature/flagd/blob/2f8e7efc69474bfd88e30ef66acfce7cdf23ccc3/flagd/main.go#L12